src/models/sensor-gdx-manager.ts

```diff
@@ -137,17 +140,28 @@ export class SensorGDXManager extends SensorManager {
       this.stopRequested = true;
     }
 
+    // Will also update live sensor display values for calibration.
     requestHeartbeat(enabled: boolean): void {
-        this.manageHeartbeat(enabled, () => {
-          if (!this.disconnectRequested) {
-            const config = cloneDeep(this.internalConfig);
-            this.enabledSensors.forEach((sensor: any, index: number) => {
-              const cNum = this.initialColumnNum + index;
-              config.columns[cNum.toString()].liveValue = sensor.value.toString();
-            });
-            this.onSensorHeartbeat(new SensorConfiguration(config));
-          }
-        });
+      if(enabled) {
+        if(this.gdxDevice) {
+          // calibration display: We only need to sample at ½ heartbeat interval
```

Comment:
Maybe move this comment up to line 11 (at the `const` declaration) and add why we only need to sample at ½ heartbeat interval